backend/tracim_backend/tests/utils.py

```diff
@@ -245,6 +246,23 @@ def last_event(self) -> typing.Optional[Event]:
         return self._session.query(Event).order_by(Event.event_id.desc()).limit(1).one()
 
 
+class DockerCompose:
+    command = [
+        "docker-compose",
```

Comment:
Docker compose should be added as requirement somewhere in test documentation.
It should also be installed in travis (already done ?)

Reply:
Enhanced the documentation for this, especially as I also use it for frontend dev/tests.